travis/travis_run_tests

```diff
@@ -31,24 +31,32 @@ def main(test_list):
 
 
 if __name__ == '__main__':
-    do_lint_check = os.environ.get("LINT_CHECK", '1') == "1"
-    do_tests = os.environ.get("TESTS", '1') == "1"
-    export_transifex = os.environ.get("TRANSIFEX") == "1"
-    # Variable TRAVIS_PULL_REQUEST corresponds to pull request number if the
-    # current job is a pull request, "false" if it's not a pull request.
-    travis_pull_request = os.environ.get("TRAVIS_PULL_REQUEST")
+    lint_check_disabled = os.environ.get('LINT_CHECK') == '0'
+    lint_check_enabled = os.environ.get('LINT_CHECK') == '1'
+    tests_enabled = os.environ.get('TESTS') == '1'
+    tests_disabled = os.environ.get('TESTS') == '0'
+    tests_unspecified = os.environ.get('TESTS') is None
+    travis_enabled = os.environ.get('TRANSIFEX') == '1'
```

Comment:
Why `travis_enabled` ? You already have `transifex_enabled` to deal with `TRANSIFEX`.
And `tests_disabled` is not used so unnecessary for the moment.